File: Data Structures & Algorithms/find-lucky-integer-in-an-array/submission-0.cpp
class Solution {
public:
    int findLucky(vector<int>& arr) {
        vector<int>res;
        int fre[500]={};
        for(int a: arr){
            fre[a]++;
        }
        for(int i=0;i<arr.size();i++){
            if(arr[i]==fre[arr[i]]){
                res.push_back(arr[i]);
            }
        }
        if(!res.empty())
        {return *max_element(res.begin(),res.end());}
        else{
        return -1;}
        
    }
};

File: Data Structures & Algorithms/sqrtx/submission-0.py
class Solution:
    def mySqrt(self, x: int) -> int:
        l,r=1,x
        while(l<=r):
            m=l+(r-l)//2
            res=m*m
            if(res>x):
                r=m-1
            elif(res<x):
                l=m+1
            else:
                return m
        return r